### DISCUSSION TITLE
P&L chart: Size meaning

### DISCUSSION THREAD

**Discussion by jjviton** [kernc/backtesting.py#898] [Q&A] [answered]
Hi all,
I´m quite new using this tool, I would like to understand the meaning of Size paramater. I use basic buy order, so I asume I expend all my money at any order.

thanks in advanced

**Comment by kernc** [comment, marked as answer]
Have you found `size=` described in [the docs](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.buy)?

**Reply by jjviton** [reply]
ahhhhh
It is the number of stocks bought.   SIZE = ActualCAsh/UnitStockPrice

**Reply by jjviton** [reply]
then, what do you represent with triangle size? it is not linked to trade size

**Reply by kernc** [reply]
Indeed, plot triangle size is proportional to trade size:
https://github.com/kernc/backtesting.py/blob/0ce24d80b1bcb8120d95d31dc3bb351b1052a27d/backtesting/_plotting.py#L400-L411

**Reply by jjviton** [reply]
thanks Kernc for the explanation.

Thank you also for the great job
